packages/core/src/validate.js

```diff
@@ -263,14 +263,45 @@ export default function validateFormData(
   };
 }
 
+function isDict(v) {
+  return (
```

Comment:
I believe we could make this a bit simpler by doing:

`return v.constructor === Object`

https://stackoverflow.com/questions/38304401/javascript-check-if-dictionary
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/constructor